stories/components/UserNameOptionRenderer.js

```diff
@@ -0,0 +1,29 @@
+import _ from 'lodash'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const UserNameOptionRenderer = (props) => {
+  const { getSelectValue, option, options, value, valueKey } = props
+  const checked = _.some(value, (val) => {
+    const valueObj = getSelectValue({ options, valueKey, value: val })
+    return valueObj[valueKey] === option[valueKey]
+  })
+  const onChange = (e) => { e.preventDefault() }
+
+  return (
+    <span>
+      <input type="checkbox" checked={checked} onChange={onChange} />
+      {`${option.lastName}, ${option.firstName}`}
```

Comment:
Hm, that's a lot to override if you just want to customize the label per option. 🤔 

Thinking we need something like an optional optionLabel and optionLabelRenderer?